src/clojurewerkz/elastisch/native/document.clj

```diff
@@ -359,31 +359,13 @@
 
 (defn scroll-seq
   "Returns a lazy sequence of all documents for a given scroll query"
-  [^Client conn prev-resp]
+  [^Client conn prev-resp & {:keys [search_type]}]
```

Comment:
Please drop the `&`. Pseudo-kwargs is considered to be a failed idea in the community. Elastisch only supports it in the public API for compatibility.